Removed invalid scopes, added example request

c4ldas opened the pull request:
- The following scopes were removed from documentation as they are invalid:
```text
session:read 
contest:read 
contest:write 
giveaway:read 
giveaway:write 
store:read 
store:write
```

- Also added an example request to show the authorization header used when doing the requests.

bgiorgio0506 commented on authentication.md:
```diff
@@ -1,2 +1,25 @@
 # Authentication
+When making a request, you must include your account ID from within stream elements. Find your Account ID <a href="https://streamelements.com/dashboard/account/channels">here</a>.
```
When making request for endpoints such as points or session you will need to send via Header a Bearer token type JWT (Json Web Token) more info <a href="https://auth0.com/docs/jwt">here</a> 
and also your personal account id as a token in the paramas

5diestel commented on authentication.md:
```diff
@@ -1,2 +1,25 @@
 # Authentication
+When making a request, you must include your account ID from within stream elements. Find your Account ID <a href="https://streamelements.com/dashboard/account/channels">here</a>.
 
+Here is an example request from <a href="http://getpostman.com"> PostMan</a>: https://api.streamelements.com/kappa/v2/channels/LIRIK?token=<account_id>
```
Specifying your account ID in the request URL doesn't work as authorization. Like @bgiorgio0506 said, you use a JWT token for authorization by sending it as Authorization header. You can retrieve your personal JWT token from your [StreamElements settings page](https://streamelements.com/dashboard/account/channels) (Show secrets)

The only time you need your channel ID is when you make requests that use channel ID instead of channel name.
So for example if you wanted to retrieve the points for a certain channel you would send a request to `https://api.streamelements.com/kappa/v2/points/CHANNEL_ID/alltime`

Pretty much all of the "important" endpoints don't even need authorization though.
You can fetch channel info from anyone and then just get their ID from that response and look up their points with it.

A better example for authorization would be:

```
curl -H 'Authorization: Bearer YOUR_JWT_TOKEN' \
-X GET 'https://api.streamelements.com/kappa/v2/overlays/YOUR_CHANNEL_ID'
```

Example response:
```json
{
    "total": 2,
    "docs": [
        {
            "settings": {
                "width": 1920,
                "height": 1080,
                "name": "1080p"
            },
            "type": "mobile",
            "preview": "https://cdn.streamelements.com/uploads/e4e348a6-5cc1-417e-966c-a9d75051da28.jpg",
            "mobile": false,
            "_id": "123",
            "channel": "12345",
            "theme": "PURPLE WAVE STATIC ALERTS",
            "themeId": "5bc59b880ffc75008af892b0",
            "game": null,
            "name": "Mobile - MOBILE ALERTS - PURPLE WAVE ALERTS STATIC",
            "createdAt": "2019-02-02T17:33:39.446Z",
            "updatedAt": "2019-02-02T17:33:39.446Z"
        },
        {
            "settings": {
                "width": 1920,
                "height": 1080,
                "name": "1080p"
            },
            "type": "regular",
            "preview": "https://static-cdn.jtvnw.net/ttv-boxart/Talk%20Shows-272x380.jpg",
            "mobile": false,
            "_id": "321",
            "updatedAt": "2017-11-15T12:03:26.202Z",
            "createdAt": "2017-08-16T18:28:42.617Z",
            "game": null,
            "name": "Followers",
            "theme": "false",
            "channel": "12345"
        }
    ]
}
```

c4ldas committed:
Removed invalid scopes, added example request

c4ldas committed:
Merge branch 'StreamElements:main' into main

c4ldas committed:
Added `channel:read` as a scope

Added channel:read as a scope

As channel:read is the default one when no scopes is chosen, I have forgotten to add it to the list.
